config/config.go

```diff
@@ -301,6 +302,42 @@ func (b *nullableBool) UnmarshalJSON(data []byte) error {
 	return err
 }
 
+// AtomicBool is a helper type for atomic operations on a boolean value.
+type AtomicBool struct {
```

Comment:
Please add some test cases for AtomicBool.

Reply:
Add some tests for ```AtomicBool``` to encode/decode toml and other tests have been implemented.